bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncTable.java

```diff
@@ -147,7 +148,9 @@ public CheckAndMutateBuilderImpl(BigtableDataClient client, HBaseRequestAdapter
         byte[] row, byte[] family) {
       this.client = client;
       this.builder = new CheckAndMutateUtil.RequestBuilder(hbaseAdapter, row, family);
-      this.requestContext = RequestContext.create(InstanceName.of("projectId", "instanceId"), "appProfileId");
+      BigtableTableName bigtableTableName = hbaseAdapter.getBigtableTableName();
```

Comment:
Please add a new method in `BigtableTableName` to create an `InstanceName`  Perhaps `.toGcbInstanceName`